ext/semian/resource.h

```diff
@@ -0,0 +1,75 @@
+/*
+For core semian resource functions exposed directly to ruby.
+
+Functions here are associated with rubyland operations.
+*/
+
+#ifndef SEMIAN_RESOURCE_H
+#define SEMIAN_RESOURCE_H
+
+#include "semian.h" // FIXME remove this once temporary declarations are removed
+#include "types.h"
+
+// Ruby variables
+ID id_timeout;
+int system_max_semaphore_count;
```

Comment:
it is required by Init_semian and semian_resource_initialize, which are in semian.c and resource.c respectively, so I don't think so.